osu.Game/Screens/Play/HUD/SkinnableModDisplay.cs

```diff
@@ -30,6 +30,9 @@ public partial class SkinnableModDisplay : CompositeDrawable, ISerialisableDrawa
         [SettingSource(typeof(SkinnableModDisplayStrings), nameof(SkinnableModDisplayStrings.ExpansionMode), nameof(SkinnableModDisplayStrings.ExpansionModeDescription))]
         public Bindable<ExpansionMode> ExpansionModeSetting { get; } = new Bindable<ExpansionMode>();
 
+        [SettingSource(typeof(SkinnableModDisplayStrings), nameof(SkinnableModDisplayStrings.DisplayDirection))]
+        public Bindable<Direction> Direction { get; } = new Bindable<Direction>();
```

Comment:
incredibly convenient that the default value of this enum happens to match the pre-existing state so user skins aren't broken